src/wallets/index.ts

```diff
@@ -55,38 +55,6 @@ export async function getWallets() {
   return wallets || [];
 }
 
-/**
- * Hook to get if there are no wallets added
- */
-export const useNoWallets = () => {
-  const [state, setState] = useState(false);
-
-  useEffect(() => {
-    (async () => {
-      const activeAddress = await getActiveAddress();
```

Comment:
Not ideal because the state won't change if the wallets are removed. Same with the `useDecryptionKey` hook below. Both have been replaced with `useWallets()` which gets the state from `<WalletsProvider>` which subscribes to state updates.